flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV1.java

```diff
@@ -95,4 +97,12 @@ public boolean moveToTrash(FileSystem fs, Path path, Configuration conf, boolean
 			throw new IOException("Failed to move " + path + " to trash", e);
 		}
 	}
+
+	@Override
+	public void alterTable(IMetaStoreClient client, String databaseName, String tableName, Table table) throws InvalidOperationException, MetaException, TException {
+		// For Hive-1.2.1, we need to tell HMS not to update stats. Otherwise, the stats we put in the table
```

Comment:
Nice!

Can you also add comments to explain how Hive-2.3.4 handles it too? Will Hive 2.3.4 just not update any table stats in parameters when client.alter_table is invoked?